live-examples/js-examples/reflect/reflect-deleteproperty.html

```diff
@@ -8,7 +8,7 @@
 console.log(object1.property1);
 // expected output: undefined
 
-var array1 = [1, 2, 3, 4, 5];
+let array1 = [1, 2, 3, 4, 5];
```

Comment:
Here also const will work. @wbamberg what do you think?

Reply:
Agreed.